test/jdk/java/lang/invoke/VarHandles/VarHandleTestAccessBoolean.java

```diff
@@ -297,26 +295,26 @@ static void testInstanceFinalField(VarHandleTestAccessBoolean recv, VarHandle vh
         // Plain
         {
             boolean x = (boolean) vh.get(recv);
-            assertEquals(x, true, "get boolean value");
+            assertEquals(true, x,"get boolean value");
```

Comment:
```suggestion
            assertEquals(true, x, "get boolean value");
```

Reply:
This is a template bug, will fix the template and regenerate the test series.